### PR TITLE
Preparation for enabling travis ci ...

### PR DESCRIPTION
**Pull request by meiliang86**
and fix an issue that cause test to hang if fork join pool only has single thread
It will be a followup task to run docker tests as well in CI.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -60,17 +60,6 @@ compileJava {
             ".*/generated-sources/.*"
 }
 
-license {
```
Comment (mfateev) [inline build.gradle]: Whey are these removed?

File: src/main/java/com/uber/cadence/internal/testservice/TestWorkflowService.java [hunk 2]
```diff
@@ -104,6 +104,8 @@
   // key->WorkflowId
   private final Map<WorkflowId, TestWorkflowMutableState> openExecutions = new HashMap<>();
 
+  private final ForkJoinPool forkJoinPool = new ForkJoinPool(4);
```
Comment (mfateev) [inline src/main/java/com/uber/cadence/internal/testservice/TestWorkflowService.java]: Why 4? Can we make it unbounded?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - SKIP_DOCKER_SERVICE=true # used to skip docker service tests in WorkflowTest for now.
+
+before_install:
+  - pushd /tmp
+  - wget http://www-us.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
+  - tar xfz thrift-0.9.3.tar.gz
+  - cd thrift-0.9.3
+  - ./configure --enable-libs=no --enable-tests=no --enable-tutorial=no --with-cpp=no --with-c_glib=no --with-java=yes --with-ruby=no --with-erlang=no --with-go=no --with-nodejs=no
+  - make -j2 && sudo make install
+  - popd
```
Comment (mfateev) [inline .travis.yml]: new line at the end
will it provide code coverage?

File: build.gradle (hunk 1 above)
Comment (meiliang86) [reply build.gradle]: These two sections are currently duplicated...See above.
Probably we had a bad merge previously.

Comment (mfateev) [reply build.gradle]: I see.

File: src/main/java/com/uber/cadence/internal/testservice/TestWorkflowService.java (hunk 2 above)
Comment (mfateev) [reply src/main/java/com/uber/cadence/internal/testservice/TestWorkflowService.java]: I think more than 1 should be OK. Let's keep it 4 for now.

**Pull request merged**